standardtracer/raw.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // RawSpan encapsulates all state associated with a (finished) Span.
 type RawSpan struct {
```

Comment:
Do not waste any time reading any of the diffs under `standardtracer/`... I made the changes I needed to in order to get a working example and nothing more :)

If we want to support the `standardtracer` idiom going forward, I would suggest that `TraceContext` move into this package... it makes a lot of things cleaner for implementations even if we decide it's not needed for OpenTracing callers.